### DISCUSSION TITLE
is there a way to avoid using scrollbars?

### DISCUSSION THREAD

**Discussion by fproverbio** [Quill-OS/quill#72] [Q&A]
i recently started to use inkbox to revive an old kobo touch 905C. when it was running the original kobo os, the built in reader would adjust how much text was displayed on a page based on the font size. With inkbox, when the font size is too large to fit all the page on the display, the remainig text is either cut off (and thus invisible) or accessible via a scroll bar. Using the scrollbar is quite tricky and requires multiple tries to succesfully scroll. Is there a way to make the reader push the cut-off text to the next page?

**Comment by Szybet** [comment]
We are working on a complete rewrite of the reader to adress this issue.

For now, install the koreader user app and use that instead
